speiseplan_projekt/src/components/Navbar.jsx

```diff
@@ -38,10 +35,10 @@ export default function Navbar() {
             <span className='font-bold font-sans text-green-500'>Kantinerado</span>
           </h1>
         </Link>
-        
       </div>
+
       {/* Login button */}
-      <Link href={"/login"}>
+      <Link href={"/profile"}> {/* Profile link */}
```

Comment:
das routet jetzt zur profile page und die middleware redirected nicht mehr zur admin page das heisst wir brauchen noch ein weg um auf die admin page zu kommen wenn ich das richtig verstehe

Reply:
fixed with backend api validation and sidebar button 
![image](https://github.com/user-attachments/assets/e29b0888-d986-4fc4-bf6d-3cd26dc25805)